marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/artifacts/core.sjs

```diff
@@ -26,7 +27,8 @@ const cachedArtifacts = {};
 const registeredArtifactTypes = {
     loadData: LoadData,
     flows: Flows,
-    stepDefinitions: StepDefs
+    stepDefinitions: StepDefs,
+    mapping: Mapping
```

Comment:
For consistency, shouldn't this be "mappings" and "Mappings"?

Reply:
@rjrudin @ryanjdew We used to have a folder like "mappings/{mappingstepname}" to be used to store [entity model <=> doc instance] mapping configuration with a filename like {mappingstepname}.{version}.mapping.json .
currently if i use "mapping" as artifact type, a new "mapping" folder willl be created to store the mapping artifacts with filename  like {mappingName}.mapping.json. If we use "mapings", it will cause name conflict. I am not sure it is a good name or a better folder stucture we should define for "mapping" operations.

Reply:
I'm confused - the directory for mappings in DHF today is "mappings" (and "flows" for flows and "step-definitions" for step definitions). Aren't these mapping artifacts still going into the "mappings" directory?

Reply:
Yes, It can.  we have {mappingName} as subfolder. i will modify it to mappings

Reply:
The dir structure today for mappings is mappings/<flowName-stepName>/<flowName-stepName-version>.mapping.json

version starts with 0 when a mapping step is created and bumped to 1 when mapping is configured after ingestion. Version remains at 1 with further changes to mapping.